modules/oe_paragraphs_media_field_storage/oe_paragraphs_media_field_storage.install

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * OE Paragraphs Media Field Storage install.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Implements hook_install().
+ */
+function oe_paragraphs_media_field_storage_install() {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs_media_field_storage') . '/config/import');
```

Comment:
First check if the field is there already, if so just exit, if not then proceed. Also you can simply get the storage for `field_storage_config` entity types and check if that entity is there.